metrics.py

```diff
@@ -67,13 +68,19 @@ def L2_frame_consistency(folder, cut_in_half=True): # cut in half: if the frame
         print("Error: check the input folder.")
         return
 
+    # Adding gaussian blur (NOTE: sigma is a hyperparam)
+    list = img_list # temp variable, so we can avoid append
```

Comment:
it would probably be better to use img_list.copy() here? Otherwise list still refers to img_list, but I think it's fine. Good point though I should definitely use append less